src/ui/home/components/identity.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { View, Text } from 'react-native'
+
+interface Props {
+  // did: string
+}
+
+interface State {}
+
+export class IdentityComponent extends React.Component<Props, State> {
+  render() {
+    return (
+      <View>
+        {/* <Text> {this.props.did} </Text> */}
+        <Text> IDENTITY </Text>
+      </View>
+    )
+  }
+}
```

Comment:
Can these be SFCs? Since we have corresponding containers.
This might be irrelevant for now and we could decide on it when we are actually implementing them.

Reply:
Yes, these will be modified in the actual implementations of the screens. They are just placeholders for now.